modular_skyrat/modules/colony_fabricator/code/machines/wall_cell_charger.dm

```diff
@@ -0,0 +1,37 @@
+/obj/machinery/cell_charger_multi/wall_mounted
+	name = "mounted multi-cell charging rack"
+	desc = "The innovative technology of a cell charging rack, but mounted neatly on a wall out of the way!"
+	icon = 'modular_skyrat/modules/colony_fabricator/icons/cell_charger.dmi'
+	icon_state = "wall_charger"
+	circuit = null
+	flags_1 = NODECONSTRUCT_1
+	max_batteries = 3
+	charge_rate = 750
+	/// The item we turn into when repacked
+	var/repacked_type = /obj/item/wallframe/cell_charger_multi
+
+/obj/machinery/cell_charger_multi/wall_mounted/Initialize(mapload)
+	. = ..()
+	AddElement(/datum/element/repackable, repacked_type, 1 SECONDS)
+	find_and_hang_on_wall()
```

Comment:
Does this actually ensure that it only gets mounted on the wall if there isn't already something on the wall?

Reply:
The wallframe item you place them with checks this for you.

Reply:
Okay good.